modules/core/src/lib/deck-picker.js

```diff
@@ -135,6 +135,9 @@ export default class DeckPicker {
     unproject3D,
     onViewportActive
   }) {
+    // picking can only hanle up to 255 items in `layers`. Drop non-pickable layers from the list.
+    layers = layers.filter(layer => this.pickLayersPass._shouldDrawLayer(layer));
```

Comment:
Make this more self documenting by introducing a temporary variable name `pickableLayers`?

Issue a warning if we exceed 255 layers, now that we are actually starting to hit this limit...?

Truncate the list to 255 to avoid  "undefined" behavior?